components/ws-daemon/pkg/iws/iws.go

```diff
@@ -766,7 +766,7 @@ func (wbs *InWorkspaceServiceServer) MountNfs(ctx context.Context, req *api.Moun
 	}
 
 	log.WithField("source", req.Source).WithField("target", req.Target).WithField("staging", nodeStaging).WithField("args", req.Args).Info("Mounting nfs")
-	cmd := exec.CommandContext(ctx, "mount", "-t", "nfs4", "-o", req.Args, req.Source, nodeStaging)
+	cmd := exec.CommandContext(ctx, "nsenter", "-n", "-t", strconv.Itoa(int(supervisorPID)), "mount", "-t", "nfs4", "-o", req.Args, req.Source, nodeStaging)
```

Comment:
> This way, we cannot mount NFS in a Docker container within the workspace, although it is a rare case.

This is what some customers are doing now. It would be breaking to land this change, if the use case for docker container is broke.

Reply:
I see, that means it should still work 😄 , I will double check.

Reply:
EFS still works with docker compose.

Presently, I'm not able to get EFS Access Points to work with Docker Compose.

Reply:
Non-blocking, suggestion:
Follow-up with @Furisto to address the above questions/uncertainties.